streaming/src/test/scala/org/apache/spark/streaming/util/WriteAheadLogSuite.scala

```diff
@@ -199,6 +198,45 @@ class FileBasedWriteAheadLogSuite
 
```

Comment:
Since some of the changes were in `FileBasedWriteAheadLog*Reader`, these should be unit tested as well. Probably add it CommonWALTests. Simple unit test that writes to the log to get WAL segments, deletes the files and then tries to read it (similar to the test "handling file errors while reading rotating logs")